hbp_knowledge/receptors/buckingham2009.bel

```diff
@@ -1164,18 +1253,21 @@ ApoE complex on PI hydrolysis were blocked by estrogen,
 and this disruption was itself blocked by wortmannin,
 suggesting that PI3K mediates estrogen’s effect
 on PI hydrolysis."
-
+SET Confidence = "Medium"
 SET CellLine="SH-SY5Y"
 
-act(a(CHEBI:carbachol)) -| a(CHEBI:phosphatidylinositol)
-p(HBP:"APOE e4") -| act(a(CHEBI:carbachol), ma(MESH:Hydrolysis))
-a(CHEBI:"amyloid-beta") -| act(a(CHEBI:carbachol), ma(MESH:Hydrolysis))
+act(a(CHEBI:carbachol)) -> act(a(CHEBI:phosphatidylinositol),ma(MESH:Hydrolysis))
```

Comment:
i think the hydrolysis of PI induced by carbachol is mentioned in this evidence. This statement would refer to the case that PI has hydrolsis activity, which is not the case as far as I know

Reply:
Then how should i indicate hydrolysis activity of carbachol?